### ISSUE TITLE
[a11y] Correct label for each activities and fragments

### ISSUE THREAD

**Issue by Aitorbp** [owncloud/android#4458]
**Is your feature request related to a problem? Please describe.**

In relation to this issue https://github.com/owncloud/android/issues/4373, we have noticed that when we have talkback enabled and we are switching between activities/fragments, talkback reads the word `ownCloud` for each change.
This is not accessible to users, since the word `ownCloud` does not provide much information to define which view the user is on.

**Describe the solution you'd like**
A possible solution would be to add the `android:label` in the `AndroidManifest.xml` in each activity to the correct name. 

In the case of fragments, we must change the label through the `setTitle`. Example below:
`activity?.setTitle("The new label here")`
Doc: https://stackoverflow.com/questions/27918701/android-fragment-change-title

### TASKS

- [x] Research (if needed)
- [x] Create branch feature/correct_label_for_each_activities_fragments
- [x] Development tasks
  - [x] Added label to activities
  - [x] Added setTitle to fragment
- [x] Code review and apply changes requested
- [x] Design test plan
- [ ] QA
- [ ] Merge branch feature/correct_label_for_each_activities_fragments
